central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -195,7 +195,7 @@ func (m *managerImpl) ProcessScanRequest(ctx context.Context, scanRequest *stora
 		err = m.updateClusterStatus(ctx, scanRequest.GetId(), clusterID, status)
 		if err != nil {
 			log.Error(err)
-			return nil, errors.Errorf("Unable to save scan configuration status for scan named %q.", scanRequest.GetScanConfigName())
+			return nil, errors.Errorf("Unable to save scan configuration status for scan named %q: %s.", scanRequest.GetScanConfigName(), err)
```

Comment:
We might need to be careful here since this could also include update failure details, not just if the cluster exists.

This is partially due to `updateClusterStatus` obfuscating database errors and existence checks behind the same function.

@dashrews78 do we want to break that apart so we don't leak errors that aren't related to existence checks?

Reply:
No, I don't think that is necessary in this case.

```suggestion
			return nil, errors.Wrapf(err, "Unable to save scan configuration status for scan named %q", scanRequest.GetScanConfigName())
```